a10_neutron_lbaas/db/models.py

```diff
@@ -53,24 +59,47 @@ def find_all(cls, db_session=None):
         return cls.query(db_session).all()
 
     @classmethod
+    def create(cls, **kwargs):
```

Comment:
Uhh, no, you don't.  Every object should have an id and timestamps. Adding the timestamps is a no-work change everywhere else, though I'm fine phasing it in later.